packages/e2e-tests/specs/editor/plugins/iframed-multiple-block-stylesheets.test.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	activatePlugin,
+	createNewPost,
+	deactivatePlugin,
+	insertBlock,
+	getEditedPostContent,
+	openDocumentSettingsSidebar,
+	clickButton,
+	canvas,
+} from '@wordpress/e2e-test-utils';
+
+async function getComputedStyle( context, property ) {
+	await context.waitForSelector(
+		'.wp-block-test-iframed-multiple-stylesheets'
+	);
+	return await context.evaluate( ( prop ) => {
+		const container = document.querySelector(
+			'.wp-block-test-iframed-multiple-stylesheets'
+		);
+		return window.getComputedStyle( container )[ prop ];
+	}, property );
+}
+
+describe( 'iframed multiple block stylesheets', () => {
+	beforeEach( async () => {
+		await activatePlugin( 'gutenberg-test-iframed-multiple-stylesheets' );
+		await createNewPost( { postType: 'page' } );
+	} );
+
+	afterEach( async () => {
+		await deactivatePlugin( 'gutenberg-test-iframed-multiple-stylesheets' );
+	} );
+
+	it( 'should load multiple block stylesheets in iframe', async () => {
+		await insertBlock( 'Iframed Multiple Stylesheets' );
+
+		expect( await getEditedPostContent() ).toMatchSnapshot();
```

Comment:
We probably don't need the snapshot here. if necessary, we can check instead whether `wp-block-test-iframed-multiple-stylesheets` class is in the document.

Reply:
Fixed